packages/core/src/sass/real-account-signup.scss

```diff
@@ -198,13 +198,15 @@
     }
 
     .proof-of-identity {
+
         &__footer {
             padding: 1.6rem 2.4rem;
             justify-content: end;
             display: flex;
             border-top: 1px solid var(--general-section-1);
             @include mobile {
                 width: 100%;
+                position:relative;
```

Comment:
we need to have a look at this as seems like eslint fix is getting skipped and it's not formatted properly